msgpack-core/src/main/java/org/msgpack/core/MessagePacker.java

```diff
@@ -442,6 +442,14 @@ public MessagePacker packDouble(double v)
         return this;
     }
 
+    private void packSmallString(String s)
+            throws IOException
+    {
+        byte[] bytes = s.getBytes("UTF-8");
```

Comment:
This involves a look up of character encoding type table. Use `MessagePack.UTF8`